### ISSUE TITLE
Can i use multiple foreign keys?

### ISSUE THREAD

**Issue by jyoungrok** [balderdashy/sails#2200]
I want to make tables like below

Table A
attribute b_1- one of the foreign key of Table B
attribute b_2- one of the foreign key of Table B

Table B
attribute b_1- one of the primary key of Table B
attribute b_2- one of the primary key of Table B

As you can see , I want to make association using multiple keys between A and B.
But i don't know how i can make 
Please comment !

**Comment by CWyrtzen** [comment]
@jyoungrok is this resolved for you?

**Comment by sailsbot** [comment]
Thanks for posting, @jyoungrok.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jyoungrok** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

jyr3357@naver.com 님께 보내신 메일 <Re: [sails] Can i use multiple foreign keys? (#2200)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------
